src/app/components/CollapsableBookList.tsx

```diff
@@ -0,0 +1,56 @@
+import * as classNames from 'classnames';
+import { take } from 'lodash';
+import * as React from 'react';
+import MediaQuery from 'react-responsive';
+
+import { Icon } from '@ridi/rsg';
+import { ConnectedInlineHorizontalBookList } from 'app/components';
+import { Book } from 'app/services/book';
+import { Expander } from 'app/services/book/components/Expander';
+
+interface CollapsableBookListProps {
+  className: string;
+  listTitle: string;
+  uiPartTitleForTracking: string;
+  books: Book[];
+}
+
+export const CollapsableBookList: React.SFC<CollapsableBookListProps> = (props: CollapsableBookListProps) =>  {
```

Comment:
아시다시피 함수 컴포넌트도 스테이트를 가질 수 있게 되어서, `SFC`는 deprecated되었습니다.
```suggestion
export const CollapsableBookList: React.FunctionComponent<CollapsableBookListProps> = (props: CollapsableBookListProps) =>  {
```

Reply:
수정했습니다 :)

Reply:
이 이슈랑 별개로 `SFC` 쓰는 다른 함수 컴포넌트 들도 한번 싹 바꾸는게 좋을 것 같네요